src/placement/jump_map.c

```diff
@@ -338,45 +231,51 @@ set_used_targets(d_list_t *down_targets, struct pool_domain *selected_dom,
  */
 static void
 get_target(struct pool_domain *curr_dom, struct pool_target **target,
-	   uint64_t obj_key, uint8_t *dom_used, struct pl_obj_layout *layout,
-	   int shard_num)
+	   uint64_t obj_key, uint8_t *dom_used, uint8_t *tgts_used,
+	   struct pl_obj_layout *layout, int shard_num)
 {
+	int			range_set;
 	uint8_t                 found_target = 0;
-	uint8_t                 top = 0;
-	uint32_t                fail_num = 0;
 	uint32_t                selected_dom;
```

Comment:
would it be less confusing if we just call it "selected" because it's for both domain and target